sync-repos.sh

```diff
@@ -7,6 +7,7 @@ GENIENLP_HEAD=`git rev-parse HEAD`
 if [ -n "${GENIENLP_VERSION}" ] && [ "${GENIENLP_VERSION}" != "${GENIENLP_HEAD}" ]; then
   git fetch
   git checkout ${GENIENLP_VERSION}
+  git pull
```

Comment:
Do not do this. `git fetch` will already fetch the remote references, and `git checkout` will switch to the right commit.

Reply:
Ok